scripts/build/verify_signed_packages.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
-_files=$*
+mkdir -p ~/.rpmdb/pubkeys
+curl -s https://packages.grafana.com/gpg.key > ~/.rpmdb/pubkeys/grafana.key
 
 ALL_SIGNED=0
 
-for file in $_files; do
```

Comment:
Bringing in the loading of the grafana key into this script is great. But I would keep the argument. It makes the script more versatile and it also makes it a bit clear from the outside (the ci script) what it is actually doing. Alternatively, if we choose to go without argument it would be a good idea to find the path for `dist/*.rpm` relative to the script so that the script works regardless of if you run it from the root or not.

Reply:
I decided to do this change since the sign_packages script had hardcoded `dist/*.rpm` so didn't see the point, but can change so that both sign and verify takes the glob as arguments.

Reply:
I think that's a good idea. Makes it a bit more clear what they do when you read the script. And also makes them less fragile (as they won't fail if you run them from the wrong folder)